### PR TITLE
Updated ReadMe file with image deployment instructions on the car,als…

### PR DESCRIPTION
**Pull request by sarahaldelame**
…o connecting the car to the internet, and communicating with the car

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,4 +135,65 @@ Go back to the first terminal and you will see:
...
+#To test the internet connection:
+$ ping 8.8.8.8
+
+#Pull the image:
+# from your car's terminal, where you previously SSH into type:
+docker pull seresearch/2018-dit-168:v0.3.5-armhf
+
+#Now that you have the container, Test that it exists by running the following command:
+docker ps
+#the command should list your pulled containers and you should find the latest one there.
```
Comment (maansthoernvik) [inline README.md]: Change container -> image

Comment (maansthoernvik) [reply README.md]: And doesn't docker image ls make more sense?

File: README.md [hunk 2]
```diff
@@ -135,4 +135,65 @@ Go back to the first terminal and you will see:
...
+#Thats for deploying.
+
+##for running an executable and communicate with the car##
+1. again from the ssh session, Clone the github repository:
+git clone https://github.com/Adam2092/DIT168Example
+# you might need to install git for the above command.
+
+#Before you can send any commands to the car you need to start the proxies. In the repository you cloned, there is a file named
+#docker-compose.yml , this file contains instruction commands for starting the proxies on the car. In order to execute these instructions, run the following:
+docker compose [docker-compose.yml]
```
Comment (maansthoernvik) [inline README.md]: docker-compose, not docker compose

Comment (maansthoernvik) [reply README.md]: You will need to build the image of the pulled repository first as well.

File: README.md [hunk 3]
```diff
@@ -135,4 +135,65 @@ Go back to the first terminal and you will see:
...
+
+#Now that you have the container, Test that it exists by running the following command:
+docker ps
+#the command should list your pulled containers and you should find the latest one there.
+
+#Thats for deploying.
+
+##for running an executable and communicate with the car##
+1. again from the ssh session, Clone the github repository:
+git clone https://github.com/Adam2092/DIT168Example
```
Comment (Rashadse) [inline README.md]: you may want to surround commands with  

 ``` 
commmand
 ``` 

like the rest of the readme file 
this way commands look distinguishable

File: README.md (hunk 2 above)
Comment (madiseniman) [reply README.md]: Why is docker compose needed? also This is the part I couldn't get to work - even though i have docker-compose installed.

**Pull request closed without merging**